app/src/main/java/com/example/rocketreserver/Apollo.kt

```diff
@@ -1,11 +1,15 @@
 package com.example.rocketreserver
 
 import android.content.Context
-import com.apollographql.apollo3.ApolloClient
 import com.apollographql.apollo3.network.okHttpClient
 import okhttp3.Interceptor
 import okhttp3.OkHttpClient
 import okhttp3.Response
+import com.apollographql.apollo3.ApolloClient
+
+val apolloClient = ApolloClient.Builder()
+    .serverUrl("https://apollo-fullstack-tutorial.herokuapp.com/graphql")
+    .build()
```

Comment:
Any reason you didn't reuse `apolloClient(context)` below? `Context` is ubiquitous in the Android world so you should be able to get one from most places

Reply:
I'm not sure I understand this comment, is there something that should change in this section of code?

Reply:
In that same file, [just below](https://github.com/apollographql/apollo-kotlin-tutorial/blob/9f9559ff5753d828600c5b49c5eaf39a32074243/app/src/main/java/com/example/rocketreserver/Apollo.kt#L16), there's a function that returns a global `ApolloClient`:

```kotlin
fun apolloClient(context: Context): ApolloClient { /*...*/ }
```

You can use `apolloClient(context)` instead of `apolloClient` everywhere and that makes sure there's only a single `ApolloClient` active in the app.

Reply:
I think I understand more then yesterday on this 😅 , but I'm still not clear on what i should change it to.. what would line 16 need to change to? Or feel free to PR this directly 😄